pkg/identityserver/storetest/api_key_store.go

```diff
@@ -267,3 +267,39 @@ func (st *StoreTest) TestAPIKeyStorePagination(t *T) {
 		}
 	})
 }
+
+// TestAPIKeyStorePaginationDefaults tests the default pagination values.
+func (st *StoreTest) TestAPIKeyStorePaginationDefaults(t *T) {
+	app1 := st.population.NewApplication(nil)
+
+	var all []*ttnpb.APIKey
+	for i := 0; i < 102; i++ {
+		_, key := st.population.NewAPIKey(app1.GetEntityIdentifiers(), ttnpb.Right_RIGHT_APPLICATION_ALL)
+		key.Name = fmt.Sprintf("Key %d", i)
+		all = append(all, key)
+	}
+
+	sort.Sort(apiKeysByID(all))
+
+	s, ok := st.PrepareDB(t).(interface {
+		Store
+
+		is.APIKeyStore
+	})
+	defer st.DestroyDB(t, false)
+	if !ok {
+		t.Skip("Store does not implement APIKeyStore")
+	}
+	defer s.Close()
+
+	t.Run("FindAPIKeys_PageLimit", func(t *T) {
+		a, ctx := test.New(t)
+
+		var total uint64
+		paginateCtx := store.WithPagination(ctx, 0, 1, &total)
```

Comment:
By default if the `page` parameter is zero it should be set to `1`, so we might as well do the test with:
```suggestion
		paginateCtx := store.WithPagination(ctx, 0, 0, &total)
```